apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStoreParallelLiveTest.java

```diff
@@ -121,7 +126,7 @@ public void uploadMultipartBlob() {
 
    @Test(dependsOnMethods = "uploadMultipartBlob", singleThreaded = true)
    public void downloadParallelBlob() throws IOException {
-      final File downloadedFile = new File(BIG_FILE.getName() + ".downloaded");
+      final File downloadedFile = new File(BIG_FILE + ".downloaded");
```

Comment:
I was going to suggest making this name random across test runs, but given the way `BIG_FILE` is created, that should already be the case? Just wanting to make sure I'm not missing something here.